src/utility/Ex3.py

```diff
@@ -0,0 +1,195 @@
+
+    while len(pedestrians) < num_agents:
+        age = random.randint(data["ageDistribution"]["distribution"]["normal"]["minValue"],
+                             data["ageDistribution"]["distribution"]["normal"]["maxValue"])
+        personality = random.choice(["open", "neurotic"])
+
+        if num_neuroticism >= 0.53 * num_agents:
+            personality = "open"
+
+        if num_openness >= 0.53 * num_agents:
+            personality = "neurotic"
+
+        if len(pedestrians) < num_personel:
+            # Personel
+            status = random.choices(["noDisabilityNoOvertaking", "noDisabilityOvertaking"],
+                                    weights=[data["walkability"]["distribution"]["noDisabilityNoOvertaking"]["value"],
+                                             data["walkability"]["distribution"]["noDisabilityOvertaking"]["value"]])[0]
+            journeyss = random.sample(journeys, 3)  # Chọn ngẫu nhiên 3 khoa viện
+            pedestrian = Personel(age, personality, status, journeyss)
+        elif len(pedestrians) < num_noDisability:
+            # Visitor
+            status = random.choices(["noDisabilityNoOvertaking", "noDisabilityOvertaking"],
+                                    weights=[data["walkability"]["distribution"]["noDisabilityNoOvertaking"]["value"],
+                                             data["walkability"]["distribution"]["noDisabilityOvertaking"]["value"]])[0]
+            journey = random.choice(journeys)
+            pedestrian = Visitor(age, personality, status, journey)
+            num_visitors += 1
+        else:
+            # Patient
+            journey = random.choice(journeys)
+            # Lấy danh sách các loại bệnh và trọng số tương ứng
+            statuses = ["crutches", "sticks", "wheelchairs", "blind"]
+            weights = [data["walkability"]["distribution"][status]["value"] for status in statuses]
+            # Chuẩn hóa trọng số để tổng bằng 1
+            total_weight = sum(weights)
+            weights = [weight / total_weight for weight in weights]
+            status = random.choices(statuses, weights=weights)[0]
+            pedestrian = Patient(age, personality, status, journey)
+            num_patients += 1
+            if status == "crutches":
+                num_crutches += 1
+            elif status == "sticks":
+                num_sticks += 1
+            elif status == "wheelchairs":
+                num_wheelchairs += 1
+            elif status == "blind":
+                num_blind += 1
+
+        if pedestrian.age < 11 and pedestrian.status == "neurotic":
+            continue
+
+        if isinstance(pedestrian, Personel):
+            if pedestrian.age < 23 or pedestrian.age > 61:
+                continue
+        if personality == "open":
+            num_openness += 1
+        elif personality == "neurotic":
+            num_neuroticism += 1
+
+        pedestrians.append(pedestrian)
+
+    print("Number of personel:", num_personel)
```

Comment:
I think printing out the results will help you check whether it's what you want, but after checking it's ok, you should delete it. Of course that's not necessary now. Please delete it!